.github/workflows/build-binary-preview.yml

```
@@ -0,0 +1,30 @@
+name: Binary preview
+
+on:
+  push:
+    branches:
+      - 'release/**'
+      - 'hotfix/**'
+
+jobs:
+  build:
+    name: Build
+    runs-on: macos-12
+    permissions:
+      contents: write
+    steps:
+      - uses: actions/checkout@v4
+
+      - name: Create, or merge into, the corresponding preview branch
+        run: git switch -c '${{ github.ref_name }}-preview' || git switch -m '${{ github.ref_name }}-preview'
```

Gah, it only occurred to me after I clicked the merge button that @nuttycom's branch name suggestion also needed to be changed here:
```suggestion
        run: git switch -c '${{ github.ref_name }}/preview' || git switch -m '${{ github.ref_name }}/preview'
```
Otherwise the `preview` suffix gets interpreted as part of the version.

---

Fixed in #163.